### PR TITLE
Use the CHROME_EXTENSION_ID environment variable

### PR DESCRIPTION
**Pull request by seanh**
The Chrome extension ID that bouncer tries to talk to was hardcoded (to
my dev extension). Instead actually read the CHROME_EXTENSION_ID
environment variable, send it to the JavaScript code, and have the
JavaScript code use it.

### CODE REVIEW

File: bouncer/scripts/test/redirect-test.js [hunk 1]
```diff
@@ -110,6 +111,24 @@ describe('redirect()', function () {
...
+  it('calls chrome.runtime.sendMessage correctly', function () {
+    window.chrome = {
+      runtime: {
+        sendMessage: sinon.stub()
+      }
+    };
+
+    redirect(function () {});
+
+    assert.equal(window.chrome.runtime.sendMessage.calledOnce, true);
```
Comment (robertknight) [inline bouncer/scripts/test/redirect-test.js]: Not going to stop merge, but I'll note that you can make this somewhat less verbose using `assert.calledWith(spy, matchers...)`

Comment (seanh) [inline bouncer/scripts/test/redirect-test.js]: The problem is the third argument is an inline function from `redirect.js` though

Comment (seanh) [inline bouncer/scripts/test/redirect-test.js]: Oh I see you could use it to test just the first two though

**Comment by robertknight** [comment]
LGTM.

File: bouncer/scripts/test/redirect-test.js (hunk 1 above)
Comment (robertknight) [inline bouncer/scripts/test/redirect-test.js]: @seanh - Sinon has a concept called [matchers](http://sinonjs.org/docs/#sinon-match-api) which allow you to provide arguments to `assert.calledWith` that are not the exact object you want to match but instead a predicate that tests whether the argument matches some condition.

**Pull request merged**